docs/features/linkerd/tutorials/linkerd-authorization-policies.mdx

```diff
@@ -0,0 +1,234 @@
+
+```yaml
+{@include: ../../../../static/code-examples/linkerd-authorization-policies/intents.yaml}
+```
+
+To apply it, use:
+```shell
+kubectl apply -n otterize-tutorial-linkerd -f ${ABSOLUTE_URL}/code-examples/linkerd-authorization-policies/intents.yaml
+```
+### See it in action
+
+<details>
+    <summary>Optional: check deployment status</summary>
+    Check that the client and server pods were deployed
+
+    ```bash
+    kubectl get pods -n otterize-tutorial-linkerd
+    ```
+
+    You should see
+
+    ```
+    NAME                           READY   STATUS    RESTARTS   AGE
+    client-68b775f766-749r4         2/2     Running   0          32s
+    nginx-c646898-2lq7l             2/2     Running   0          32s
+    other-client-74cc54f7b5-9rctd   2/2     Running   0          32s
+    ```
+
+</details>
+
+monitor both client attempts to call the server with additional terminal windows,
+so we can see the effects of our changes in real time.
+
+2. **Open a new terminal window [client]** and tail the client log:
+
+```bash
+kubectl logs -f --tail 1 -n otterize-tutorial-linkerd deploy/client
+```
+
+<details>
+    <summary>Expected output</summary>
+
+    At this point the client should be able to communicate with the server:
+
+    ```
+    Calling server...
+    HTTP/1.1 200 OK
+    ...
+    hello from /client-path
+    ```
+
+</details>
+
+3. **Open another terminal window [client-other]** and tail the other-client log:
+
+```bash
+kubectl logs -f --tail 1 -n otterize-tutorial-linkerd deploy/other-client
+```
+
+<details>
+    <summary>Expected output</summary>
+
+    At this point the client should be able to communicate with the server:
+
+    ```
+    Calling server...
+    HTTP/1.1 200 OK
+    ...
+    hello from /other-client-path
+    ```
+
+</details>
+
+Keep an eye on the logs being tailed in the **[other-client]** terminal window,
+and apply this `intents.yaml` file in your **main terminal window** using:
+
+```shell
+kubectl apply -f ${ABSOLUTE_URL}/code-examples/linkerd-authorization-policies/intents.yaml
+```
+
+:::tip
+Client intents are the cornerstone of [intent-based access control (IBAC)](/overview/intent-based-access-control).
+::: 2. You should quickly see in the **[other-client]** terminal that it times out when calling the server,
+as expected since it didn't declare its intents:
+
+```bash
+Calling server...
+HTTP/1.1 200 OK
+...
+hello from /other-client-path  # <- before applying the intents file
+# highlight-start
+Calling server...              # <- after applying the intents file
+curl timed out
+Terminated
+# highlight-end
+```
+
+3. And in the **[client]** terminal you should see that calls go through, as expected since they were declared:
+
+```bash
+Calling server...
+HTTP/1.1 200 OK
+...
+hello from /client-path
+```
+
+4. You should also see that a new Linkerd authorization policy was created:
+
+```bash
+kubectl get authorizationpolicies.policy.linkerd.io -n otterize-tutorial-linkerd
```

Comment:
Show + explain here the rest of the resources and their contents. It's less straightforward than Istio, so we need to show and explain it.

Reply:
Please also show the resources themselves and explain what Otterize did, because it's not obvious like in Istio